cli/src/main/java/com/adaptivescale/rosetta/cli/model/Config.java

```diff
@@ -1,13 +1,20 @@
 package com.adaptivescale.rosetta.cli.model;
 
 import com.adaptivescale.rosetta.common.models.input.Connection;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.List;
 import java.util.Optional;
 
 public class Config {
     private List<Connection> connections;
 
+    @JsonProperty("OpenAI_API_KEY")
```

Comment:
Maybe it is better if we name this: `openai_api_key`
and for the Java part use `openAIApiKey`

Reply:
Yes i agree, this alongside with the model will be renamed